coralproject/talk
# [CORL-2895] make sure broken image urls preview for flair badges

## What does this PR do?

Make sure broken image urls preview for flair badges.

Needed to make sure the alt text was a space so it would actually show the broken image if the url wasn't loading. Without an alt text, chrome hides broken images and won't show the broken image icon.

## These changes will impact:


## What changes to the GraphQL/Database Schema does this PR introduce?

None

## Does this PR introduce any new environment variables or feature flags?

No

## If any indexes were added, were they added to `INDEXES.md`?

None

## How do I test this PR?

- Go to `Config > General > Custom flair badges`
- Add a flair badge with a broken URL
    - using `https://somethingsomething.com/posts/img.jpeg` should work
- Add the flair badge
- See it shows a broken image url like so

<img width="849" alt="image" src="https://github.com/coralproject/talk/assets/5751504/e3eaf809-06e6-4e3c-ae06-9d616979c106">

## Where any tests migrated to React Testing Library?

No

## How do we deploy this PR?

No special considerations.
